packages/dds/tree/src/simple-tree/core/treeNodeSchema.ts

```diff
@@ -438,19 +438,23 @@ export enum NodeKind {
 	/**
 	 * A node which serves as a map, storing children under string keys.
 	 */
-	Map,
+	Map = 0,
```

Comment:
Note for reviewers: just made the existing values here explicit. In the event that more changes are needed here, this makes it a bit more clear that the values should be preserved.